plugins/gcp/fix_plugin_gcp/resources/compute.py

```diff
@@ -111,6 +124,8 @@ class GcpAddress(GcpResource):
         "status": S("status"),
         "subnetwork": S("subnetwork"),
         "users": S("users", default=[]),
+        "ip_address": S("address", default=""),
```

Comment:
```suggestion
        "ip_address": S("address"),
```
let's make the base class property optional. 
Empty string as default is a lie.